adapters/AppLovin/AppLovinAdapter/RTB/GADMAppLovinRtbInterstitialRenderer.m

```diff
@@ -0,0 +1,118 @@
+//
+//  GADMAppLovinRtbInterstitialRenderer.m
+//  Adapter
+//
+//  Created by Christopher Cong on 7/17/18.
+//  Copyright © 2018 Google. All rights reserved.
+//
+
+#import "GADMAppLovinRtbInterstitialRenderer.h"
+#import "GADMAdapterAppLovinUtils.h"
+#import "GADMAdapterAppLovinConstant.h"
+#import "GADMAdapterAppLovinExtras.h"
+
+#import <AppLovinSDK/AppLovinSDK.h>
+
+@interface GADMAppLovinRtbInterstitialRenderer () <GADMediationInterstitialAd, ALAdLoadDelegate, ALAdDisplayDelegate, ALAdVideoPlaybackDelegate>
+
+/// Data used to render an RTB interstitial ad.
+@property (nonatomic, strong) GADMediationInterstitialAdConfiguration *adConfiguration;
+
+/// Callback object to notify the Google Mobile Ads SDK if ad rendering succeeded or failed.
+@property (nonatomic, copy) GADInterstitialRenderCompletionHandler renderCompletionHandler;
+
+/// Delegate to notify the Google Mobile Ads SDK of interstitial presentation events.
+@property (nonatomic, strong, nullable) id<GADMediationInterstitialAdEventDelegate> adEventDelegate;
+
+/// Controlled Properties
+@property (nonatomic, strong) ALSdk *sdk;
+@property (nonatomic, strong) ALInterstitialAd *interstitialAd;
+@property (nonatomic, strong) ALAd *ad;
+
+@end
+
+@implementation GADMAppLovinRtbInterstitialRenderer
+
+- (instancetype)initWithAdConfiguration:(GADMediationInterstitialAdConfiguration *)adConfiguration
+                      completionHandler:(nonnull GADInterstitialRenderCompletionHandler)handler {
+    self = [super init];
+    if (self) {
+        self.adConfiguration = adConfiguration;
+        self.renderCompletionHandler = handler;
+        
+        self.sdk = [GADMAdapterAppLovinUtils retrieveSDKFromCredentials:adConfiguration.credentials.settings];
```

Comment:
Are you sure this is `serverParameters`? Android is diff.

Reply:
Yep, AdMob have a note for it `/// Mediation configuration set by the publisher on the AdMob frontend.` and it's the only option for retrieving parameters.